db/migration/migrations/000009_metadata_table.up.sql

```diff
@@ -0,0 +1,116 @@
+BEGIN;
+
+DROP VIEW IF EXISTS endpoint_export;
+
+DROP TABLE IF EXISTS fhir_endpoints_metadata;
+
+CREATE TABLE IF NOT EXISTS fhir_endpoints_metadata (
+    id                      SERIAL PRIMARY KEY,
+    url                     VARCHAR(500),
+    http_response           INTEGER,
+    availability            DECIMAL(5,4),
+    errors                  VARCHAR(500),
+    response_time_seconds   DECIMAL(7,4),
+    smart_http_response     INTEGER,
+    created_at              TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    updated_at              TIMESTAMPTZ NOT NULL DEFAULT NOW()
+);
+
+ALTER TABLE fhir_endpoints_info
+DISABLE TRIGGER add_fhir_endpoint_info_history_trigger;
+
+ALTER TABLE fhir_endpoints_info 
+ADD COLUMN metadata_id INT REFERENCES fhir_endpoints_metadata(id) ON DELETE SET NULL;
+
+ALTER TABLE fhir_endpoints_info_history 
+ADD COLUMN metadata_id INT REFERENCES fhir_endpoints_metadata(id) ON DELETE SET NULL;
+
+
+CREATE OR REPLACE FUNCTION populate_endpoints_metadata_info() RETURNS VOID as $$
+    DECLARE
+        t_curs cursor for select * from fhir_endpoints_info;
+        t_row fhir_endpoints_info%ROWTYPE;
+        j INTEGER;
+    BEGIN
+        FOR t_row in t_curs LOOP
+            INSERT INTO fhir_endpoints_metadata (url, http_response, availability, errors, response_time_seconds, smart_http_response, created_at, updated_at) VALUES (t_row.url, t_row.http_response, t_row.availability, t_row.errors, t_row.response_time_seconds, t_row.smart_http_response, t_row.created_at, t_row.updated_at);
```

Comment:
I don't think this line is needed since the same information would get added from the history table (see image)
![Screen Shot 2021-01-22 at 3 01 59 PM](https://user-images.githubusercontent.com/54954141/105553678-d79a4a80-5cd3-11eb-9fea-26952fe4793c.png)

Reply:
The only issue with getting rid of that line, is that I am not sure how I would insert the correct metadata id into the info table, as there is no distinctive way grab the correct info_history entry that the current endpoint info entry corresponds to. The only thing I can think of, and I can change it to work like this if you both think this would be better, is to go through the endpoint info table after populating the metadata table from the history table and grab the most recent entry's metadata_id in the info history table for each endpoint url in the info table. Let me know what you guys think!

Reply:
I think it makes sense to do it that way, if it's not too complicated!

Reply:
Made this change!